tap_jira/streams.py

```diff
@@ -146,11 +146,49 @@ def sync(self):
 
 class BoardsGreenhopper(Stream):
     def sync(self):
+        # BOARDS endpoint
         if Context.is_selected(BOARDS.tap_stream_id):
             path = "/rest/greenhopper/1.0/rapidview"
             boards = Context.client.request(self.tap_stream_id, "GET", path)['views']
             self.write_page(boards)
 
+        if Context.is_selected(VELOCITY.tap_stream_id):
+            for board in boards:
+                # VELOCITY endpoint
+                boardId = str(board["id"])
+                path = ("/rest/greenhopper/1.0/rapid/charts/velocity.json?rapidViewId=" + boardId)
+                # get data from the Velocity endpoint
+                velocity = Context.client.request(VELOCITY.tap_stream_id, "GET", path)
+
+                if velocity.get("sprints"):
+                    sprintData = velocity["sprints"]
+
+                    # per Sprint in the Sprint-section of the data, add the Board id, Estimated value & Completed value from the VelocityStatEntries-section
+                    for sprint in sprintData:
+                        sprintId = str(sprint["id"])
+                        velocityStats = {
+                            "boardId": board["id"],
+                            "velocityEstimated": velocity["velocityStatEntries"][sprintId]["estimated"]["value"],
+                            "velocityCompleted": velocity["velocityStatEntries"][sprintId]["completed"]["value"]
+                            }
+                        sprint.update(velocityStats)
+                    VELOCITY.write_page(sprintData)
+
+                # SPRINTS endpoint
+                if (Context.is_selected(SPRINTS.tap_stream_id) and board["sprintSupportEnabled"]):
```

Comment:
Is there a reason this cant be run without being dependent on velocity? I may be missing something while reviewing, but it looks like you can run the Sprints stream without needing to turn on the Velocity stream.  Is that correct?